src/components/shared/form_fields/CheckboxField.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+	<div class="form-field form-field-checkbox">
+		{{ fieldModel }}
```

Comment:
this output should probably not be there

Reply:
Good spot, looks like I forgot about a few debug thingies..